index.html

```diff
@@ -61,14 +73,36 @@
                 <li>Famliy Time is a must.</li>
               </ul>
           </section>
-        </section>
+          <section id='bio'>
+            <h2>Bio</h2>
+            <p>Hello there, thank you for checking out my work space. My name is Dan Stineback and I'm a web developer....well aspiring web developer. Web development wasn't always on my radar but has always been something that I loved doing. So I decided to take the leap and become a full time developer. This site is a work in progress but in the coming months will be launched live and out into the world. Until then I hope you enjoy it.</p>
+            </section>
+        </article>
+      </section>
+      <section id="resume" class="tab-content">
+
+           <p>Here is my Resume!</p>
+
+           <div ="section"><object data="img/dan_stineback_resume.pdf" type="application/pdf" width="100%" height="500px;"></object></div>
```

Comment:
If you have the time, consider writing up and HTML version of your resume instead of just embedding a PDF.